app/core/recipe_models.py

```diff
@@ -0,0 +1,12 @@
+import uuid
+
+from django.db import models
+
+
+class Recipe(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+    ingredients = models.JSONField()
```

Comment:
i don't think it's necessary for this exercise, but if you want to extract ingredients to their own entity, you can do it like so in django:

```
class Ingredient(models.Model):
    name = models.CharField(max_length=255)
    recipe = models.ForeignKey(Recipe, related_name="ingredients", on_delete=models.CASCADE)
    ```
and then you would be able to access the ingredients from a recipe through `recipe.ingredients`

Reply:
For the exercise is completely valid but in a real-world scenario I'd avoid adding JSON fields in relational databases unless it really makes sense. [This answer](https://stackoverflow.com/a/15367769) on SO summarizes what I think on the topic. My rule of thumb is if the data is somehow "primary" to my application I would always consider modeling all the fields in the DB (when using relational DBs)